generator/config/stage/setWindowFields.yaml

```diff
@@ -28,3 +28,117 @@ arguments:
         description: |
             Specifies an expression to group the documents. In the $setWindowFields stage, the group of documents is known as a partition. Default is one partition for the entire collection.
         optional: true
+tests:
+    -
+        name: 'Use Documents Window to Obtain Cumulative Quantity for Each State'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/setWindowFields/#use-documents-window-to-obtain-cumulative-quantity-for-each-state'
+        pipeline:
+            -
+                $setWindowFields:
+                    partitionBy: '$state'
+                    sortBy:
+                        orderDate: 1
+                    output:
+                        cumulativeQuantityForState:
+                            $sum: '$quantity'
+                            window:
+                                documents: ['unbounded', 'current']
```

Comment:
We need an enum for `unbound` / `current`...

Reply:
Should this be ticketed under [PHPLIB-1381](https://jira.mongodb.org/browse/PHPLIB-1381)?

Reply:
I already have this list of enums in [`expression.php`](https://github.com/mongodb/mongo-php-builder/blob/c0daa64a75590465d6c052ed78b60b31814bff72/generator/config/expressions.php#L121-L162). I don't feel the need to create a ticket for each.

Reply:
Noted. I missed those `@todo` comments.